insights-notification-schemas-java/src/main/resources/schemas/Action.json

```diff
@@ -127,11 +124,25 @@
   ],
   "anyOf": [
     {
+      "type": "object",
+      "properties": {
+        "account_id": {
+          "type": "string",
+          "title": "Account id"
+        }
+      },
       "required": [
         "account_id"
       ]
     },
     {
+      "type": "object",
+      "properties": {
+        "account_id": {
+          "type": [ "string", "null" ],
+          "title": "Account id"
+        }
+      },
```

Comment:
if `org_id` is required - `account_id` has to be a string or null (or not defined at all)